docs/core-study-details.md

```diff
@@ -232,32 +256,54 @@ for more information.
 
 ### core__encounter
 
-|        Column        |   Type    |Description|
-|----------------------|-----------|-----------|
-|id                    |varchar    |           |
-|enc_class_code        |varchar(6) |           |
-|enc_class_display     |varchar(21)|           |
-|status                |varchar    |           |
-|type_code             |varchar    |           |
-|type_code_system      |varchar    |           |
-|sevicetype_code       |varchar    |           |
-|sevicetype_code_system|varchar    |           |
-|priority_code         |varchar    |           |
-|priority_code_system  |varchar    |           |
-|reasoncode_code       |varchar    |           |
-|reasoncode_code_system|varchar    |           |
-|age_at_visit          |bigint     |           |
-|start_date            |date       |           |
-|end_date              |date       |           |
-|start_week            |date       |           |
-|start_month           |date       |           |
-|start_year            |date       |           |
-|subject_ref           |varchar    |           |
-|encounter_ref         |varchar    |           |
-|gender                |varchar    |           |
-|race_display          |varchar    |           |
-|ethnicity_display     |varchar    |           |
-|postalcode3           |varchar    |           |
+|             Column             |   Type    |Description|
+|--------------------------------|-----------|-----------|
+|id                              |varchar    |           |
+|status                          |varchar    |           |
+|enc_class_code                  |varchar(6) |           |
+|enc_class_display               |varchar(21)|           |
```

Comment:
What causes these columns to be size-limited? We probably don't want them to be, yeah?